backend/src/documentation/java/wooteco/prolog/docu/StudylogDocumentation.java

```diff
@@ -154,14 +155,17 @@ class StudylogDocumentation extends Documentation {
         회원과_멤버그룹_그룹멤버를_등록함();
         String studylogLocation1 = 스터디로그_등록함(createStudylogRequest1()).header("Location");
         String studylogLocation2 = 스터디로그_등록함(createStudylogRequest2()).header("Location");
+        String studylogLocation3 = 스터디로그_등록함(createStudylogRequest3()).header("Location");
         Long studylogId1 = Long.parseLong(studylogLocation1.split("/studylogs/")[1]);
         Long studylogId2 = Long.parseLong(studylogLocation2.split("/studylogs/")[1]);
+        Long studylogId3 = Long.parseLong(studylogLocation3.split("/studylogs/")[1]);
 
         스터디로그_단건_조회(studylogId1);
         스터디로그_단건_조회(studylogId2);
+        스터디로그_단건_조회(studylogId3);
```

Comment:
c: 조금 더 네이밍을 명확하면 어떨까요?

```suggestion
        String frontStudylogLocation = 스터디로그_등록함(createFrontStudylogRequest()).header("Location");
        String backStudylogLocation = 스터디로그_등록함(createBackStudylogRequest()).header("Location");
        String androidStudylogLocation = 스터디로그_등록함(createAndroidStudylogRequest()).header("Location");
        Long frontStudylogId = Long.parseLong(frontendStudylogLocation.split("/studylogs/")[1]);
        Long backStudylogId = Long.parseLong(backendStudylogLocation.split("/studylogs/")[1]);
        Long androidStudylogId = Long.parseLong(androidStudylogLocation.split("/studylogs/")[1]);
        
        스터디로그_단건_조회(frontStudylogId);
        스터디로그_단건_조회(backStudylogId);
        스터디로그_단건_조회(androidStudylogId);
```

Reply:
수정했습니다!
나머지 테스트 코드는 리팩터링 진행하면서 테스트 코드도 수정해보려고 해요~